presto-main/src/main/java/com/facebook/presto/server/remotetask/ContinuousTaskStatusFetcher.java

```diff
@@ -208,7 +208,7 @@ public void fatal(Throwable cause)
         }
     }
 
-    synchronized void updateTaskStatus(TaskStatus newValue)
```

Comment:
We had this to protect `lastRequest` that we used to have. I missed taking it out when I removed `lastRequest`.